src/pybind/mgr/dashboard/frontend/src/app/core/layouts/login-layout/login-layout.component.scss

```diff
@@ -29,4 +29,9 @@
     min-height: 100vh;
     width: 100vw;
   }
+
```

Comment:
This can be discussed and implemented later, but how about changing this to white?
the contrast of the blue with the background is not the best and seems worst with more text.
![image](https://user-images.githubusercontent.com/399326/93216584-71b31a80-f757-11ea-8ad4-a8ea726ae858.png)

![image](https://user-images.githubusercontent.com/399326/93216503-59db9680-f757-11ea-9b47-85ff73d91fa9.png)

Reply:
I agree. The hovering over links makes it look more bad with the background.

Reply:
I've changed the colors and updated the screenshot in description.